functions/go/create-setters/README.md

```diff
@@ -7,20 +7,19 @@
 Add setter comments to matching resource fields. Setters serve as
 parameters for the template-free setting of field values.
 
-Setters are a safer alternative to other substitution techniques which do not
-have the context of the structured data. Setter comments can be added to
-parameterize the field values of resources using this function.
+Setters are a safer alternative to parameterize field values as they
+have the context of the structured data. This function helps you with
+adding setter comments to field values you intend to parameterize.
```

Comment:
Can you define setters once (maybe in apply-setters) instead of repeating it everywhere. What if you you have other setter-related functions in the future?

Reply:
Done.